http_server.py

```diff
@@ -483,36 +480,38 @@ async def file_post_handler(request: web_request.Request) -> web.Response:
 
 async def submit_vote_handler(request: web_request.Request) -> web.Response:
     vote_input = await request.json()
-    
+
+    print(vote_input)
```

Comment:
Did this make it into this commit by accident?

Reply:
aye, I didn't trust the debuger.